src/components/Dashboard/Header/Authentication/Signup.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { Component } from 'react';
+import {
+  Form, Button, Container, Row, Col,
+} from 'react-bootstrap';
+import { Redirect } from 'react-router-dom';
+import { Toaster, Intent } from '@blueprintjs/core';
+import db from '../../../../fire';
+import NavBar from '../Header';
+
+export default class Signup extends Component {
+  constructor(props) {
```

Comment:
That's no big deal but FYI, `props` is not a required parameter in a constructor. If you don't use props in a constructor, you may simply write it like this:
```
constructor() {
  super()
  this.state = { redirect: false };
}
```